apps/fluent-tester/src/FluentTester/theme/ThemePickers.macos.tsx

```diff
@@ -32,6 +34,37 @@ type PartPickerProps = {
   onChange: (value: string) => void;
 };
 
+const ThemeMenuPicker = ({ selected, onChange, collection, style }) => {
+  const chevronXml = `
+  <svg width="12" height="16" viewBox="0 0 11 6" color="#000">
+    <path fill='currentColor' d='M0.646447 0.646447C0.841709 0.451184 1.15829 0.451184 1.35355 0.646447L5.5 4.79289L9.64645 0.646447C9.84171 0.451185 10.1583 0.451185 10.3536 0.646447C10.5488 0.841709 10.5488 1.15829 10.3536 1.35355L5.85355 5.85355C5.65829 6.04882 5.34171 6.04882 5.14645 5.85355L0.646447 1.35355C0.451184 1.15829 0.451184 0.841709 0.646447 0.646447Z' />
+  </svg>`;
+
+  return (
+    <View style={{ flexDirection: 'row', alignItems: 'center', margin: 5, ...style }}>
```

Comment:
Out of curiosity why can't we use MenuPicker here?

Reply:
Oh it's probably because it's outside of TestComponents. Okay.

Reply:
We could probably do a relative import `import { MenuPicker } from '../TestComponents/Common' if we wanted to. It looks like `ThemePickers.tsx` is already doing that. It doesn't look like we can delete `ThemePickers.macos.tsx` right now, but the relative import should be fine?

Context:
When I worked on bringing FluentTester to mobile, I found that Picker on iOS didn't work well with FluentTester's layout: It was a giant carousel instead of a button with a menu. That was an issue unique to iOS picker (Android, Windows, and macOS didn't have this problem). We also didn't want the light/dark option overridden in the apple theme because PlatformColr stuff. Sooo.. I made ThemePicker platform specific. 

 but we can just change this to be a real